### PR TITLE
[dcmtk] no absolute paths

### PR DESCRIPTION
**Pull request by autoantwort**
For https://github.com/microsoft/vcpkg/issues/20469

### CODE REVIEW

File: scripts/cmake/vcpkg_fixup_pkgconfig.cmake [hunk 1]
```diff
@@ -104,7 +104,7 @@ function(vcpkg_fixup_pkgconfig)
             string(REPLACE "${unix_packages_dir}" [[${prefix}]] contents "${contents}")
             string(REPLACE "${unix_installed_dir}" [[${prefix}]] contents "${contents}")
 
-            string(REGEX REPLACE "(^|\n)prefix[\t ]*=[^\n]*" "" contents "${contents}")
+            string(REGEX REPLACE "(^|\n) *prefix[\t ]*=[^\n]*" "" contents "${contents}")
```
Comment (FrankXie05) [inline scripts/cmake/vcpkg_fixup_pkgconfig.cmake:107]:
```
            string(REGEX REPLACE "(^|\n) *prefix[\t ]*=[^\n]*" "" contents "${contents}")
```
For regular expressions, the `*` sign is greedy, can you explain it in detail? :)

**Review by FrankXie05 — CHANGES_REQUESTED** [review]

File: scripts/cmake/vcpkg_fixup_pkgconfig.cmake (hunk 1 above)
Comment (dg0yt) [reply scripts/cmake/vcpkg_fixup_pkgconfig.cmake:107]: The `*` applies to the space character in front of it: Match any sequence of space characters, here: at the beginning of the line.

Comment (FrankXie05) [reply scripts/cmake/vcpkg_fixup_pkgconfig.cmake:107]: For whitespace paths, I don't think it's allowed. 🤔

Comment (dg0yt) [reply scripts/cmake/vcpkg_fixup_pkgconfig.cmake:107]: > For whitespace paths, I don't think it's allowed. thinking

It is valid and widely used. Plain regex.

**Review by FrankXie05 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks! Should we run another test of your tool PR now?

**Comment by autoantwort** [comment]
That would be nice. https://github.com/microsoft/vcpkg/pull/24845 and https://github.com/microsoft/vcpkg/pull/22363 are still missing.

**Pull request merged**